src/plugins/presentation_util/public/components/field_picker/field_picker.tsx

```diff
@@ -23,13 +23,15 @@ import {
 } from '@elastic/eui';
 import { DataView, DataViewField } from '@kbn/data-views-plugin/common';
 
+import type { GetCustomFieldType } from '@kbn/field-utils';
 import { FieldTypeFilter } from './field_type_filter';
 
 import './field_picker.scss';
 
 export interface FieldPickerProps {
   dataView?: DataView;
   selectedFieldName?: string;
+  getCustomFieldType?: GetCustomFieldType<DataViewField>;
```

Comment:
Hi @viajes7,

Thanks for noticing and working on a fix! 

Looks like `getCustomFieldType` is not being used in a parent component. I think we can remove the associated changes.

Reply:
@jughosta fixed, Thanks for reviewing the code, `getCustomFieldType` removed

Reply:
Thanks!